### PR TITLE
Translate some `ROOT::Fit` tutorials referenced in the manual to Python

### PR DESCRIPTION
**Pull request by guitargeek**
The combinedFit.C tutorial is a good example for the `ROOT::Fitter` functionality, so we also want to have it in Python. The fitNormSum.C tutorial explains a feature that is often asked about in the forum.

### CODE REVIEW

File: tutorials/fit/fitNormSum.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+f_sum = ROOT.TF1("fsum", fnorm_exp_cb, -5.0, 5.0, fnorm_exp_cb.GetNpar())
+
+# III.:
+parameter_values = fnorm_exp_cb.GetParameters()
+f_sum.SetParameters(parameter_values.data())
+# Note: in the C++ tutorial, the parameter value sync is done in one line with:
+# ```C++
+#     f_sum->SetParameters(fnorm_exp_cb->GetParameters().data());
+# ```
+# However, GetParameters() returns in std::vector by value, which doesn't
```
Comment (lmoneta) [inline tutorials/fit/fitNormSum.py:52]: Thank you for commenting on this.
In C++ GetParameters returns a pointer to an array (double *) stored in TF1. In Python it returns a copy as an std::vector that will be deleted?

**Review by lmoneta — APPROVED** [review]
Thank you Jonas for this update in the tutorials!

File: tutorials/fit/fitNormSum.py (hunk 1 above)
Comment (guitargeek) [reply tutorials/fit/fitNormSum.py:52]: No, in case of the `TF1NormSum` it always returns a vector by value also in C++: hist/hist/inc/TF1NormSum.h

Comment (guitargeek) [reply tutorials/fit/fitNormSum.py]: I have now mentioned in the inline comment that this is specifically `TF1NormSum::GetParameters()`.

**Pull request merged**